packages/cyberstorm/src/components/componentStyles/Tag.module.css

```diff
@@ -0,0 +1,38 @@
+/* Common Tag styles */
+.tag {
+  display: flex;
+  flex: none;
+  flex-direction: row;
+  gap: var(--size-tiny);
+  align-items: center;
+  justify-content: center;
+
+  padding: var(--size-tiny) var(--size-xs);
+  border-radius: var(--size-m);
+  color: var(--text-color);
+  background-color: var(--bg-color);
+
+  --text-color: var(--color-text--default);
+  --bg-color: var(--color-surface--3);
+}
+
+.tag:hover {
+  --bg-color: var(--color-surface--4);
+}
+
+/* Styles for Tag's text */
+.tagLabel {
+  font-weight: 700;
+  font-size: var(--text-xxs);
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  line-height: var(--text-xs);
+
+  text-transform: uppercase;
+}
+
+/* Styles for Tag's Icons */
+.tagIcon {
```

Comment:
What if we want a bigger tag with icons? Should the size be a separate property that would affect size, padding, font-size etc? I guess @gkurck would be a better authority on this than me.

Reply:
Sure, wouldn't hurt having the sizes recurring throughout the designs as a prop. One can also set specific edge case stylings for a component in the parent component they are used in if needed. That's one of the main reasons I'm advocating giving each of the component's element a class to hook on.

Regarding this one, I would suggest creating the base component with the same size as it's in the component library, not with the smaller size used in the mod card.

Reply:
Refactored to match the design and use em

Reply:
Well there's still no size property on the component, e.g. `<Tag size="huge" label="Hello" />` but I'm not sure if it's required to fill the DoD? Having a "Refs TS-1234" in the commit messages would be nice to track the changes to the ticket. Couldn't find it using the ticket system's search either.